libraries/Microsoft.Bot.Builder.Dialogs/Recognizer.cs

```diff
@@ -29,14 +31,23 @@ public class Recognizer
         [JsonProperty("id")]
         public string Id { get; set; }
 
+        /// <summary>
+        /// Gets or sets the currently configured <see cref="IBotTelemetryClient"/> that logs the LuisResult event.
+        /// </summary>
+        /// <value>The <see cref="IBotTelemetryClient"/> being used to log events.</value>
+        [JsonIgnore]
+        public IBotTelemetryClient TelemetryClient { get; set; } = new NullBotTelemetryClient();
```

Comment:
>public IBotTelemetryClient TelemetryClient { get; set; } = new NullBotTelemetryClient() [](start = 8, length = 87)

Do we have to add properties for this?

Reply:
The TelemetryClient isn't currently added to TurnState.  I saw your suggestion regarding having the middleware add it, but that telemetry middleware is currently optional and we want to ensure this telemetry lights up for folks who already have it enabled when they start to adopt adaptive.

Reply:
This also keeps the telemetry implementation consistent across the SDK.  Any developers who are currently using telemetry may be using the telemetry client to log trace / custom events etc.  It's better, in my opinion, to keep that consistent so they can continue to use the same approach.